### ISSUE TITLE
solo to deploy sourcify

### ISSUE THREAD

**Issue by jeromy-cannon** [hashgraph/solo#1066]

**Comment by jeromy-cannon** [comment]
we are removing our version and using sourcify.dev a public/common version.

**Comment by Reccetech** [comment]
I don't think this is needed. The user of sourcify is the Hiero mirror node explorer. In the older architecture we had our own Sourcify instance, but recent we have migrated to the main sourcify.dev instance: https://hedera.com/blog/smart-contract-verification-sourcify-dev-now-supported/

I do not believe sourcify.dev will support Solo testing as it requires a chainID. Moreover I don't see this as a big use case for Solo. If users want to test SC verification then Testnet is a better option where sourcify.dev is supported. 

So I think we can close this ticket.
